manifest/receiver.py

```diff
@@ -18,6 +21,10 @@ def run(self):
         self.logger.info("Running ManifestReceiver")
         super(ManifestReceiver, self).run()
 
+    def notify_state_tracker(self, body_dict):
+        self.publish_config.send_message(self.producer, body_dict)
+        self.logger.info("Notified!")
```

Comment:
Do we need this message? Could some informative details be added?

Reply:
Unknown, this is the manifest receiver rather than the experiment receiver so is older code that I'm unsure how it is used.
I simply moved the method so it was clearer that it would work with the new QueueConfig publish method.